common/app/services/S3.scala

```diff
@@ -15,7 +15,6 @@ import conf.Configuration
 import model.PressedPageType
 import org.joda.time.{DateTime, DateTimeZone}
 import play.api.libs.ws.{WSClient, WSRequest}
-import sun.misc.BASE64Encoder
```

Comment:
Is this related to the Java 11 change?

Reply:
Yep! The import was actually unused as well (so the code compiles without it) but `sun.misc.BASE64Encoder` was an [internal, unsupported part of the JDK](https://www.oracle.com/java/technologies/faq-sun-packages.html), and went away with Java 9.